src/signer/ain-wallet-signer.ts

```diff
@@ -0,0 +1,19 @@
+import {Signer} from "./signer";
+
+export class AinWalletSigner implements Signer {
+  private ainetwork: Signer;
+  constructor() {
```

Comment:
minor) could add some empty lines between member variable definitions and method definitions?

Reply:
Done!